packages/client/src/components/add-liquidity/add-liquidity-v3.tsx

```diff
@@ -227,6 +227,93 @@ export const AddLiquidityV3 = ({
         return { baseTokenCurrency, quoteTokenCurrency, uniPool };
     };
 
+    const getBoundPricesAndTicks = (indicators: { [indicatorName: string]: LiquidityBand }) => {
+        const {
+            baseTokenCurrency,
+            quoteTokenCurrency,
+            uniPool,
+        } = getUniSDKInstances();
+
+        (window as any).uni = { baseTokenCurrency, quoteTokenCurrency, uniPool };
+        (window as any).bounds = bounds;
+
+        debug.indicators = indicators;
+
+        const indicator = indicators[SELECTED_INDICATOR_NAME];
+        const [lowerBound, upperBound] = indicator.bounds[sentiment];
+
+        const lowerBoundNumerator = ethers.utils
+            .parseUnits(
+                new BigNumber(lowerBound).toFixed(
+                    baseTokenCurrency.decimals
+                ),
+                baseTokenCurrency.decimals
+            )
+            .toString();
+
+        const lowerBoundDenominator = ethers.utils
+            .parseUnits('1', quoteTokenCurrency.decimals)
+            .toString();
+
+        // Convert to lower tick and upper ticks
+        const lowerBoundPrice = new Price(
+            baseTokenCurrency,
+            quoteTokenCurrency,
+            lowerBoundNumerator,
+            lowerBoundDenominator
+        );
+
+        (window as any).lowerBoundPrice = lowerBoundPrice;
```

Comment:
fyi for next time we have `debug` now so you dont have to cast `import { debug } from 'util/debug'` and you'll find everything in console under the `debug` or `d` var